docs/en/Configuration-Settings.md

```diff
@@ -8,44 +8,89 @@ priority: 1
-Note that, configuration set through environment variables may not be realized by applications.
-3. [Property files](#property-files). This is a general approach to customize any
-[supported Alluxio configuration](#appendix). Configuration in those files can be respected by Alluxio servers,
-as well as applications.
+## Alluxio Shell Commands
 
-The priority to load configuration settings, from the highest to the lowest, is
-application settings (if any), environment variables, property files and the defaults.
+Alluxio shell users can use JVM system property `-Dproperty=value` to specify an Alluxio property in commandline. For example, the following Alluxio shell command sets the write type to
+`CACHE_THROUGH` when copying files to Alluxio:
+
+```bash
+$ bin/alluxio fs -Dalluxio.user.file.writetype.default=CACHE_THROUGH copyFromLocal README.md /README.md
+```
+
+
+## Spark Jobs
+
+Spark users can pass JVM system properties to Spark jobs by adding `"-Dproperty=value"` to
+`spark.executor.extraJavaOptions` for Spark executors and `spark.driver.extraJavaOptions` for
+Spark drivers. For example, the following Spark job sets write type to `CACHE_THROUGH` when
+writing to Alluxio:
+
+```bash
+$ spark-submit \
+--conf 'spark.driver.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+--conf 'spark.executor.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+...
+```
 
-### Application settings
 
-Alluxio shell users can use `-Dkey=property` to specify an Alluxio configuration value in commandline. For example,
+## Hadoop MapReduce Jobs
 
-{% include Configuration-Settings/specify-conf.md %}
+Hadoop MapReduce users can add `"-Dproperty=value"` in `hadoop jar` command-lines. For example, the
+following MapReduce job of wordcount sets write type to `CACHE_THROUGH` when writing to Alluxio:
 
-Spark users can add `"-Dkey=property"` to `${SPARK_DAEMON_JAVA_OPTS}` in `conf/spark-env.sh`, or add it to
-`spark.executor.extraJavaOptions` (for Spark executors) and `spark.driver.extraJavaOptions` (for Spark drivers).
+```bash
+$ bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.jar wordcount \
+-Dalluxio.user.file.writetype.default=CACHE_THROUGH \
+-libjars {{site.ALLUXIO_CLIENT_JAR_PATH}} \
+<INPUT FILES> <OUTPUT DIRECTORY>
+```
+
+
+## General Java Applications
+
+For a general application using Alluxio client, one can create a
+[Java property file](https://en.wikipedia.org/wiki/.properties) named
+`alluxio-site.properties` and append the directory of this property file to the application
+classpath. Note that, this requires the privilege to set application classpath before launching
+the application, and also to deploy the property file on every node running the application.
+
+For example one can make the following change for Spark or Hadoop cluster:
+
+```bash
+$ edit /path/alluxio-site.properties
+$ export SPARK_CLASSPATH=${SPARK_CLASSPATH}:/path # for Spark jobs
```

Comment:
Does this work for the spark executors? This only affects the current local machine, right?